src/docs/stan-reference/distributions.tex

```diff
@@ -258,7 +260,31 @@ \subsection{Evaluating Vectorized Functions}
   y[n] ~ normal(mu[n], sigma);
 \end{stancode}
 
+\subsection{Evaluating Vectorized PRNG Functions}\label{vectorization-prng.subsection}
+
+Some PRNG functions accept vector arguments.  Such functions are
+indicated by the plural pseudo-types \code{reals} or \code{ints}.  As
+in the vectorized arguments subsections of
+Section~\ref{prob-vectorization.section}, the pseudo type \code{reals}
```

Comment:
You don't need to use it here, but there's a macro that lets you do `\refsec{prob-vectorization}`.  It inserts the `.section` markup and the `section~` bit.